docs/reference-hw/ad-computers.md

```diff
@@ -1,5 +1,18 @@
 # AD Computers
 
+## Advantech AD Computers
+
+![Advantech AD Computers](https://github.com/user-attachments/assets/0798fc0d-e799-4c54-8f81-dfe98466e726)
```

Comment:
Could we resize the image so that it would be a similar scale with other images?

```suggestion
<img src="https://github.com/user-attachments/assets/0798fc0d-e799-4c54-8f81-dfe98466e726" alt="Advantech AD Computers" width="50%">
```